### ISSUE TITLE
Applying a theme is buggy

### ISSUE THREAD

**Issue by khanspers** [PathVisio/pathvisio4-ant#40]
When applying a theme via Tools>Apply Theme, the theme is actually applied right away, before the pop-up warning that asks the user to confirm. Choosing No (to not apply the theme) results in the pathway theme changing back. So it works correctly, but it gives a buggy impression and it seems like the theme should not be applied until the user has confirmed?

**Comment by Finterly** [comment]
I edited the message. Does this look better? 
![image](https://user-images.githubusercontent.com/28988687/186623173-eeb6cc3c-7482-44f1-869f-6ba11ce5f8d3.png)

**Comment by khanspers** [comment]
Yes, that message works better with what is actually happening. But in general it seems not standard to apply the change before the message. What do others think? @mkutmon @DeniseSl22
